Source/Internal/Views/VirtusizeProductImageView.swift

```diff
@@ -119,7 +119,7 @@ internal class VirtusizeProductImageView: UIView {
             return nil
         }
         return URL(string:
-            "https://res.cloudinary.com/virtusize/image/upload/t_product-large-retina-v1/\(cloudinaryPublicId).jpg"
+            "https://res.cloudinary.com/virtusize/image/upload/q_auto,f_auto/\(cloudinaryPublicId).jpg"
```

Comment:
you should provide the exact size that you need to reduce the bandwidth, that can be done by adding [`w_XX,h_XX`](https://cloudinary.com/documentation/image_transformations#img_trans_overview) to the query

You can use [`dpr_auto`](https://cloudinary.com/documentation/image_optimization#responsive_image_sizing) to ensure that images are still optimized for the device

Reply:
@zr0z Please see the change in the commit 244edf7. Thanks!